packages/adsum-map/src/controllers/SelectionController.js

```diff
@@ -22,13 +23,29 @@ class SelectionController {
         return this;
     }
 
-    async selectPoi(poi: Poi, reset: boolean = true, centerOn: boolean = true): Promise<void> {
+    async selectPoi(
+        poi: Poi,
+        reset: boolean = true,
+        centerOn: boolean = true,
+        centerOnOptions: ?CameraCenterOnOptions = null,
+        stayOnCurrentFloor: boolean = true,
+        ground: ?AdsumObject3D = null,
```

Comment:
`ground: ?AdsumObject3D = this.awm.sceneManager.getCurrentFloor()`

Reply:
`this.awm` is not accessible in SelectionActions.js